util/cron/test-perf.gpu-ex-rocm.bash

```diff
@@ -10,23 +10,17 @@ source $UTIL_CRON_DIR/common-native-gpu-perf.bash
 # CONFIG_NAME
 source $UTIL_CRON_DIR/common-perf.bash
 
-# everything we source above will end up sourcing `common.bash` which will then
-# source `load-base-deps.bash`. In the system we run this config,
-# `load-base-deps.bash` ends up exporting
-#   `CHPL_LLVM_CONFIG=(which # llvm-config)`
-# If `rocm` module is loaded, rocm's llvm-config takes precedence over our LLVM
-# install. We don't want that in this system. So `module load rocm` should
-# appear after all the `source`s.
-module load rocm/5.4.3  # pin to rocm 5.4.3
+module load rocm # load the default version of ROCm
 
 export CHPL_COMM=none
-export CHPL_LLVM=system
+export CHPL_LLVM=bundled
```

Comment:
This is mostly because us not having our system LLVM built with AMDGCN, is that right?

Reply:
No, this is because we only support CHPL_GPU=amd with CHPL_LLVM=bundled due to LLVM bugs upstream